addons/event_booth_sale/models/event_booth_category.py

```diff
@@ -37,6 +37,7 @@ def _compute_price(self):
             if category.product_id and category.product_id.list_price:
                 category.price = category.product_id.list_price + category.product_id.price_extra
 
+    @api.depends_context('pricelist', 'quantity')
```

Comment:
"quantity" is something from the context?

Reply:
It seems so ^^ Yannick did this change ...